src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -394,34 +404,38 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
             </Row.Value>
           </Row>
         </Row>
-        <Row variant='gutter'>
-          <Row.Label>{translate('common.slippage')}</Row.Label>
-          <Row.Value>
-            <Skeleton isLoaded={!isQuoteDataLoading}>
-              <Amount.Crypto
-                value={quoteData?.slippageCryptoAmountPrecision ?? ''}
-                symbol={asset.symbol}
-              />
-            </Skeleton>
-          </Row.Value>
-        </Row>
-        <Row variant='gutter'>
-          <Row.Label>
-            <HelperTooltip label={translate('defi.modals.saversVaults.timeToBreakEven.tooltip')}>
-              {translate('defi.modals.saversVaults.timeToBreakEven.title')}
-            </HelperTooltip>
-          </Row.Label>
-          <Row.Value>
-            <Skeleton isLoaded={!isQuoteDataLoading}>
-              {translate(
-                `defi.modals.saversVaults.${
-                  bnOrZero(quoteData?.daysToBreakEven).eq(1) ? 'day' : 'days'
-                }`,
-                { amount: quoteData?.daysToBreakEven ?? '0' },
-              )}
-            </Skeleton>
-          </Row.Value>
-        </Row>
+        {!isRunePool ? (
```

Comment:
q: Is RUNEPool confirmed to be 0-slippage when depositing

Reply:
Yep, we've no way to configure it at all

Reply:
Asked 9R, added a todo in [19db6af](https://github.com/shapeshift/web/pull/7440/commits/19db6afed2d77fec1f3cb46386d3487d42dbb41e)